<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="combo_strat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Random File</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dynamic Hashing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+ Tree</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Estrategia de indexación</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Buscar</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar DNI</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_edit_val">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Insertar</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>DNI</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insert_dni">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="insert_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insert_nombres">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insert_apellidos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insert_edad">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de nacimiento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insert_fecha">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dd/mm/aaaa</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Solo el B+ Tree soporta búsqueda por rango</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>611</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Resultados</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>551</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>DNI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Edad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha nac.</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProyecto_BD2">
    <property name="title">
     <string>Proyecto BD2</string>
    </property>
   </widget>
   <addaction name="menuProyecto_BD2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
